tests/MessageSpec/Message.hs

```diff
@@ -37,4 +37,6 @@ programExec =
     it "calls a function with a named and an unnamed argument" $
       "module a(b,c){echo(b+c);}a(b=1,1);" --> oneMessage TextOut (SourcePosition 1 15 []) "2.0"
 --    it "warns about a missing argument" $
---      "module a(b){echo(b);}a();" --> oneMessage TextOut (SourcePosition 1 13 []) "1.0"
+--      "module a(b){echo(b);}a();" --> oneMessage TextOut (SourcePosition 1 13 []) "1.0"=
```

Comment:
how did this equal sign get here?

Reply:
Probably errant typing in vim. Boo-urns.

Reply:
Removed